environment.py

```diff
@@ -286,22 +286,22 @@ def env() -> Mapping[str, str | None]:
                 'is_custom': False
             },
             'bigquery_emulator': {
-                'ref': 'docker.io/ucscgi/azul-bigquery-emulator:0.4.4-48',
+                'ref': 'docker.io/ucscgi/azul-bigquery-emulator:0.4.4-49',
```

Comment:
Please include my changes towards https://github.com/DataBiosphere/azul-private/issues/305

```suggestion
                'ref': 'docker.io/ucscgi/azul-bigquery-emulator:0.4.4-50',
```

And please connect this PR to that issue.

Reply:
https://github.com/DataBiosphere/azul-private/issues/305#issuecomment-3599480002